ffi/src/scan.rs

```diff
@@ -351,13 +352,14 @@ pub struct CTransforms {
 pub unsafe extern "C" fn get_transform_for_row(
     row: usize,
     transforms: &CTransforms,
-) -> Option<Handle<SharedExpression>> {
+) -> OptionalValue<Handle<SharedExpression>> {
```

Comment:
changed function signature but not test change points to missing coverage? perhaps let's track an issue to fix?